src/OpenTelemetry.Instrumentation.ServiceFabricRemoting/README.md

```diff
@@ -0,0 +1,172 @@
+# Service Fabric Remoting Instrumentation for OpenTelemetry .NET
+
+| Status        |           |
+| ------------- |-----------|
+| Stability     |  |
+| Code Owners   |  |
+
+[![NuGet version badge](https://img.shields.io/nuget/v/OpenTelemetry.Instrumentation.ServiceFabricRemoting)](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.ServiceFabricRemoting)
+[![NuGet download count badge](https://img.shields.io/nuget/dt/OpenTelemetry.Instrumentation.ServiceFabricRemoting)](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.ServiceFabricRemoting)
+[![codecov.io](https://codecov.io/gh/open-telemetry/opentelemetry-dotnet-contrib/branch/main/graphs/badge.svg?flag=unittests-Instrumentation.ServiceFabricRemoting)](https://app.codecov.io/gh/open-telemetry/opentelemetry-dotnet-contrib?flags[0]=unittests-Instrumentation.ServiceFabricRemoting)
+
+This is an [Instrumentation Library](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/glossary.md#instrumentation-library),
+which instruments [Service Fabric Remoting](https://learn.microsoft.com/en-us/azure/service-fabric/service-fabric-reliable-services-communication-remoting)
+and collects telemetry about incoming requests.
+
+## Steps to enable OpenTelemetry.Instrumentation.ServiceFabricRemoting
+
+An example project is available in the
+[examples//serviceFabricRemoting](../../examples/serviceFabricRemoting/) folder.
```

Comment:
It is not true. I am not sure if you need an example. Or you can have cool enough README here.

Reply:
removed